app/controllers/thredded/topics_controller.rb

```diff
@@ -109,7 +109,10 @@ def topic_params
         .merge(user: thredded_current_user)
     end
 
-    def new_topic_params
+    def new_topic_params(allow_empty: false)
+      if allow_empty && params[:topic].blank?
+        return { messageboard: messageboard, user: thredded_current_user }
+      end
       params
         .require(:topic)
```

Comment:
Let's just replace this `require(:topic)` with `fetch(:topic, {})`, making the `allow_empty` param unnecessary.